requirements.txt

```
@@ -3,6 +3,6 @@ geopy>=1.14.0
 docopt
 pyyaml
 certifi
-hbmqtt
-websockets<=8.1
+amqtt
```

```suggestion
amqtt>=0.10.0,<0.11.0
```
I would suggest to require at least `0.10.0`. Looking at the changelog, this seems to be a good starting point. The upper bound is just in case any more additional breaking changes are introduced.